### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30427]
#### VFS team name
Public Websites

#### VFS product name
ORDMI Harassment Reporting Tool - Landing Page

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
The alert box does not use the newly updated design system <alert> which includes a `sr-only` alt text for the warning icon for screen reader users. 

### Link, screenshot or steps to recreate
[Reference ticket 19674 for more information.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19674)

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
[Use the design system's warning alert](https://design.va.gov/components/alertboxes).

### VFS Team Tasks to Complete

- [x] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by DanielleThierryUSDSVA** [comment]
We're using the component that's in the Drupal template and can't change that for launch. This is a component issue. @laflannery It looks like you're tracking this already. Thanks!

**Comment by laflannery** [comment]
@DanielleThierryUSDSVA, I am looking at [this page](https://www.va.gov/report-harassment/) (which I think it the correct page?) and don't currently see an alert being used. Was this perhaps referring to an older iteration and therefore this doesn't apply anymore to the production page?

If that's the case I am going to go ahead and close this but let me know if there is something I am missing or misunderstanding

**Comment by DanielleThierryUSDSVA** [comment]
Hi @laflannery. Yes, that's exactly it! Thanks so much for checking. Feel free to close.
